### DISCUSSION TITLE
calculate the estimated network space

### DISCUSSION THREAD

**Discussion by Nazarikov** [Chia-Network/chia-blockchain#10106] [Q&A] [answered]
Hello! What formula is used to calculate the estimated network space? Does it use: 'sub_slot_iters': 'total_iters': 'weight':
Is it possible to use these values to calculate the estimated network space? Thank you!

**Comment by mariano54** [comment, marked as answer]
Take a look at this method https://github.com/Chia-Network/chia-blockchain/blob/main/chia/rpc/full_node_rpc_api.py#L452

**Reply by Nazarikov** [reply]
Thanks, but I saw it. This is not what I need, I would like to understand the calculation formula itself.

**Reply by Nazarikov** [reply]
No, it looks like what I was looking for :)
Thanks!
